### PR TITLE
Remove the ffmpeg rtp sender

### PR DESCRIPTION
**Pull request by jerzywilczek**
This still needs some cleanup work (mainly removing ffmpeg and rtp types from the rest of the compositor). I'll do that in subsequent PRs

### CODE REVIEW

**Review by wkozyra95 — APPROVED** [review]

**Review by noituri — APPROVED** [review]
👌

**Comment by wkozyra95** [comment]
new-api was merged to master, rebase this PR

File: src/rtp_sender.rs [hunk 1]
```diff
@@ -27,35 +33,82 @@ impl PipelineOutput for RtpSender {
...
+            return;
+        };
+
+        let Some(pts) = packet.pts() else {
+            warn!("No pts is present in a packet received from the encoder");
+            return;
+        };
+
+        let data = Bytes::copy_from_slice(data);
+        let payloads = match context.payloader.payload(1500, &data) {
```
[the remark this answers is not in the document]
Comment (WojciechBarczynski) [reply src/rtp_sender.rs:77]: Maybe define const for that?

File: src/rtp_sender.rs [hunk 2]
```diff
@@ -27,35 +33,82 @@ impl PipelineOutput for RtpSender {
...
-            (*(*stream.as_mut_ptr()).codecpar).codec_id = codec::Id::H264.into();
+        if codec.id() != codec::Id::H264 {
+            unimplemented!("Only H264 is supported");
         }
 
-        output_ctx
-            .write_header()
+        let mut rng = rand::thread_rng();
+        let ssrc = rng.gen::<u32>();
+        let next_sequence_number = rng.gen::<u16>();
```
Comment (WojciechBarczynski) [inline src/rtp_sender.rs:42]: Do we have to start with a random sequence number? AFAIK RFC recommends this, but I don't see any benefits from starting at a random number

**Review by WojciechBarczynski — APPROVED** [review]

File: src/rtp_sender.rs (hunk 2 above)
Comment (jerzywilczek) [reply src/rtp_sender.rs:42]: I don't see any harm from doing that either. It's very straight-forward to do and doesn't make the implementation slower and the rfc recommends it. Why would we not do it?

**Pull request merged**